### PR TITLE
databases/week3

### PR DESCRIPTION
**Pull request by Rasim-Deniz**

### CODE REVIEW

File: databases/week3/hyf_homework_databases_lesson3.sql [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+USE hyf_meal_sharing;
+
+CREATE TABLE `meal` (
+`id` int unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,
+`title` varchar(255) NOT NULL,
+`description` text NOT NULL,
+`location` varchar(255) NOT NULL,
+`when` datetime NOT NULL,
+`max_reservations` int unsigned NOT NULL,
+`price` decimal NOT NULL,
```
Comment (martamatos) [inline databases/week3/hyf_homework_databases_lesson3.sql]: It's a better practice to specify the number of significant digits that are stored and the number of digits that can be stored following the decimal point: https://dev.mysql.com/doc/refman/8.0/en/fixed-point-types.html

File: databases/week3/hyf_homework_databases_lesson3.sql [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+insert into review (title, description, meal_id, stars, created_date) 
+	values ('Nice', 'Meal was quite good.', 5, 3, '2021-04-22');
+
+SELECT *
+FROM meal
+WHERE price < 90;
+
+SELECT meal.title, meal.max_reservations, reservation.number_of_guests, reservation.created_date
+FROM meal
+JOIN reservation ON
```
Comment (martamatos) [inline databases/week3/hyf_homework_databases_lesson3.sql]: What about meals without reservations yet?

Also, once you address that, you might want to take a look at this function: COALESCE() function: https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_coalesce (if you have meals without reservations you'll have nulls).

File: databases/week3/hyf_homework_databases_lesson3.sql [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+CREATE TABLE `meal` (
+`id` int unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,
+`title` varchar(255) NOT NULL,
+`description` text NOT NULL,
+`location` varchar(255) NOT NULL,
+`when` datetime NOT NULL,
+`max_reservations` int unsigned NOT NULL,
+`price` decimal NOT NULL,
+`created_date` date NOT NULL
+);
```
Comment (martamatos) [inline databases/week3/hyf_homework_databases_lesson3.sql]: You forgot to specify the character set and the engine, see here for an example: https://github.com/HackYourFuture-CPH/databases/blob/master/lesson2/lesson2-data.sql

Same for the other two tables.

**Review by martamatos — CHANGES_REQUESTED** [review]
Great work! I left a few comments, but nothing serious :)

File: databases/week3/hyf_homework_databases_lesson3.sql (hunk 1 above)
Comment (Rasim-Deniz) [reply databases/week3/hyf_homework_databases_lesson3.sql]: Good to know thanks a lot!

File: databases/week3/hyf_homework_databases_lesson3.sql (hunk 2 above)
Comment (Rasim-Deniz) [reply databases/week3/hyf_homework_databases_lesson3.sql]: I tried to fix it, did I get it right?

Comment (martamatos) [reply databases/week3/hyf_homework_databases_lesson3.sql]: Yep! Looks good to me, well done! :)

**Pull request merged**